### PR TITLE
Bug fixes

### LINKED ISSUE #6: Add null checks in ToString calls
To avoid throwing exceptions and interrupting workflows

### LINKED ISSUE #8: GetValueFromSource should try to get value from RunExtensionMethods too

### PR DESCRIPTION
**Pull request by alelom**
## Description

A few bug fixes.

Closes #8 
Closes #6

### CODE REVIEW

**Comment by al-fisher** [comment]
Thanks @alelom

File: CIH_Engine/Query/ValueFromSource.cs [hunk 1]
```diff
@@ -75,6 +75,11 @@ private static object GetValue(this IBHoMObject obj, string sourceName)
                     }
 
                 }
+                else
+                {
+                    // Try extracting the property using an Extension method.
+                    BH.Engine.Reflection.Compute.RunExtensionMethod(obj, sourceName);
```
Comment (al-fisher) [inline CIH_Engine/Query/ValueFromSource.cs:81]:
```suggestion
                    return BH.Engine.Reflection.Compute.RunExtensionMethod(obj, sourceName);
```

We need a `return` there 😄

**Review by al-fisher — CHANGES_REQUESTED** [review]

File: CIH_Engine/Query/ValueFromSource.cs (hunk 1 above)
Comment (alelom) [reply CIH_Engine/Query/ValueFromSource.cs:81]: Sorry, forgot to commit it!

**Review by al-fisher — APPROVED** [review]
Thanks @alelom 

This is working now

**Pull request merged**